### ISSUE TITLE
Create Cmake installer

### ISSUE THREAD

**Issue by w-klijn** [multiscale-cosim/TVB-NEST#1]
Current installation is a combination of install scripts.

Consolidate into a single Cmake installation environment that can be used on all targets:

- [ ] determine all dependencies
- [ ] Create standard file structure
- [ ] Find module for each standard dependencies
- [ ] check installation on all targets

**Comment by w-klijn** [comment]
Estimation: 
Cmake experience: 60Hours
No experience: : 120 Hours

**Comment by maedoc** [comment]
Does this mean the CMake will be building the whole Python stack from source as well?

**Comment by w-klijn** [comment]
Nope, the Python stack is a dependency that should not be install manually as part of TVB-NEST.
It should be findable via find-python3 or you ad a -DPython-path argument
